types/bun/README.md

```diff
@@ -0,0 +1,39 @@
+# TypeScript types for Bun
```

Comment:
Note that these READMEs are not actually included in the published package. Perhaps we can change that, but that's not how the infra currently works and many packages actually put contributing docs in these files.